src/views/Create.vue

```diff
@@ -11,6 +11,7 @@ import { useTerms } from '@/composables/useTerms';
 import { PROPOSAL_QUERY } from '@/helpers/queries';
 import validations from '@snapshot-labs/snapshot.js/src/validations';
 import { clone } from '@/helpers/utils';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Can we remove this line? or any reason to import it? 🤔  I see it is not used in this file

Reply:
Done